include/openthread/instance.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (384)
+#define OPENTHREAD_API_VERSION (386)
```

Comment:
Jumping over API version `385` and increasing by 2? 😃  

(this wont help avoid a rebase if `385` is used by another PR since we still get a conflict on this PR in such a case and then PR cannot be merged)

Reply:
aha, you got me, my another pr is 385, so I was thinking maybe I can increase it based on that one. Will update back!